demisto_sdk/__main__.py

```diff
@@ -47,6 +45,8 @@
 from demisto_sdk.commands.upload.upload import upload_content_entity
 from demisto_sdk.utils.utils import check_configuration_file
 
+logger = logging.getLogger("demisto-sdk")
```

Comment:
didn't we agree on a single logger, imported into the different modules? Or is it not possible?

Reply:
This will be done on phase-2.